src/core/server/models/user/user.ts

```diff
@@ -522,38 +522,54 @@ export async function insertUser(
   }
 
   // Guard against empty login profiles (they need some way to login).
-  if (input.profiles.length === 0) {
-    throw new Error("users require at least one profile");
+  if (input.profiles.length !== 1) {
+    throw new Error("users require one profile");
```

Comment:
Was saying "users require one profile" intended? This implies that they can have only exactly 1 profile and not multiples.

Unsure if that was the intention of this message.